src/es6-module-utils.ts

```diff
@@ -0,0 +1,444 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import traverse, {NodePath} from 'babel-traverse';
+import * as babel from 'babel-types';
+import * as clone from 'clone';
+import {Analyzer, FileRelativeUrl, PackageRelativeUrl, ResolvedUrl} from 'polymer-analyzer';
+import {rollup} from 'rollup';
+
+import {getAnalysisDocument} from './analyzer-utils';
+import {getNodePath, getNodeValue, parseModuleFile, rewriteNode, serialize} from './babel-utils';
+import {AssignedBundle, BundleManifest} from './bundle-manifest';
+import {Bundler} from './bundler';
+import {ensureLeadingDot, getFileName} from './url-utils';
+import {camelCase} from './utils';
+
+/**
+ * Looks up and/or defines the unique name for an item exported with the given
+ * name in a module within a in a bundle.
```

Comment:
within a bundle

Reply:
Fixed in commit 19426736259d90ffe86f2cd712cd749d50f9e2ff